### PR TITLE
Wrap NetworkAreaDiagram.getDisplayedVoltageLevels

### PR DESCRIPTION
**Pull request by tadam50**
**Please check if the PR fulfills these requirements**

- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?**
Feature

**What is the current behavior?**
No access to NetworkAreaDiagram.getDisplayedVoltageLevels

**What is the new behavior (if this is a feature change)?**
User access to NetworkAreaDiagram.getDisplayedVoltageLevels

### CODE REVIEW

File: cpp/src/bindings.cpp [hunk 1]
```diff
@@ -481,6 +481,9 @@ PYBIND11_MODULE(_pypowsybl, m) {
     m.def("get_network_area_diagram_svg", &pypowsybl::getNetworkAreaDiagramSvg, "Get network area diagram SVG as a string",
           py::arg("network"), py::arg("voltage_level_ids"), py::arg("depth"), py::arg("high_nominal_voltage_bound"), py::arg("low_nominal_voltage_bound"), py::arg("edge_name_displayed"));
 
+    m.def("get_network_area_diagram_displayed_voltageLevels", &pypowsybl::getNetworkAreaDiagramDisplayedVoltageLevels, "Get network area diagram displayed voltage level",
+          py::arg("network"), py::arg("voltage_level_ids"), py::arg("depth")));
```
Comment (So-Fras) [inline cpp/src/bindings.cpp]: Extra parenthesis to remove after "depth" --> `...arg("depth"));`

File: cpp/src/pypowsybl.cpp [hunk 2]
```diff
@@ -783,6 +783,13 @@ std::string getNetworkAreaDiagramSvg(const JavaHandle& network, const std::vecto
     return toString(callJava<char*>(::getNetworkAreaDiagramSvg, network, voltageLevelIdPtr.get(), voltageLevelIds.size(), depth, highNominalVoltageBound, lowNominalVoltageBound, edgeNameDisplayed));
 }
 
+std::vector<std::string> getNetworkAreaDiagramDisplayedVoltageLevels(const JavaHandle& network, const std::vector<std::string>& voltageLevelIds, int depth) {
+    ToCharPtrPtr voltageLevelIdPtr(voltageLevelIds);
+    auto displayedVoltageLevelIdsArrayPtr = callJava<array*>(::getDisplayedVoltageLevels, network, voltageLevelIdPtr.get(), voltageLevelIds.size(), depth);
```
Comment (So-Fras) [inline cpp/src/pypowsybl.cpp]: `(::getNetworkAreaDiagramDisplayedVoltageLevels, network, ...`

File: java/src/main/java/com/powsybl/python/network/NetworkAreaDiagramUtil.java [hunk 3]
```diff
@@ -93,6 +93,10 @@ static VoltageLevelFilter getNominalVoltageFilter(Network network, List<String>
         return new VoltageLevelFilter(voltageLevels);
     }
 
+    public static List<String> getDisplayedVoltageLevels(Network network, List<String> voltageLevelIds, int depth) {
```
Comment (So-Fras) [inline java/src/main/java/com/powsybl/python/network/NetworkAreaDiagramUtil.java:103]: `getNetworkAreaDiagramDisplayedVoltageLevels(Network network, List<String> voltageLevelIds, int depth)`

File: java/src/main/java/com/powsybl/python/network/NetworkCFunctions.java [hunk 4]
```diff
@@ -963,6 +963,16 @@ public static CCharPointer getNetworkAreaDiagramSvg(IsolateThread thread, Object
         });
     }
 
+    @CEntryPoint(name = "getNetworkAreaDiagramDisplayedVoltageLevels")
+    public static PyPowsyblApiHeader.ArrayPointer<CCharPointerPointer> getDisplayedVoltageLevels(IsolateThread thread, ObjectHandle networkHandle, CCharPointerPointer voltageLevelIdsPointer,
```
Comment (So-Fras) [inline java/src/main/java/com/powsybl/python/network/NetworkCFunctions.java]: `getNetworkAreaDiagramDisplayedVoltageLevels(IsolateThread thread...)`

File: java/src/main/java/com/powsybl/python/network/NetworkCFunctions.java [hunk 5]
```diff
@@ -963,6 +963,16 @@ public static CCharPointer getNetworkAreaDiagramSvg(IsolateThread thread, Object
...
+                                                        int voltageLevelIdCount, int depth, ExceptionHandlerPointer exceptionHandlerPtr) {
+        return doCatch(exceptionHandlerPtr, () -> {
+            Network network = ObjectHandles.getGlobal().get(networkHandle);
+            List<String> voltageLevelIds = toStringList(voltageLevelIdsPointer, voltageLevelIdCount);
+            return createCharPtrArray(NetworkAreaDiagramUtil.getDisplayedVoltageLevels(network, voltageLevelIds, depth));
+        });
```
Comment (So-Fras) [inline java/src/main/java/com/powsybl/python/network/NetworkCFunctions.java:946]: `getNetworkAreaDiagramDisplayedVoltageLevels(network, ...)`

**Review by So-Fras — CHANGES_REQUESTED** [review]
Add also:
`def get_network_area_diagram_displayed_voltageLevels(network: JavaHandle, voltage_level_ids: Union[str, List[str]], depth: int) -> List[str]: ...`
in _pypowsybl.pyi

I did this review by including your changes in the #632 PR and overwriting the diagram related versions in the pom (all of this locally of course)

**Review by So-Fras — APPROVED** [review]

**Review by geofjamg — CHANGES_REQUESTED** [review]
Documentation is missing

**Review by EtienneLt — APPROVED** [review]

**Pull request merged**